### PR TITLE
Change to PR status.

### PR DESCRIPTION
**Pull request by skynavga**

### CODE REVIEW

File: spec/status.xml [hunk 1]
```diff
@@ -19,72 +19,24 @@ href="mailto:public-tt@w3.org">public-tt@w3.org</loc> (<loc
...
-<p>A cumulative summary of all changes applied to this version since
-the current (TTML1, 2nd Edition) Recommendation was published is
-available at <loc href="ttml2-changes.html">Timed Text Markup Language
-2 (TTML2) Change Summary</loc>.  An abbreviated list of changes
-affecting language syntax can be found at
+with a subject line starting with <code>[ttml2]</code>.</p>
+
+<p>An implementation report is available
+at <loc href="https://www.w3.org/wiki/TTML/TTML2ImplementationReport">implementation
+report</loc> documenting that, for each feature introduced in this
```
Comment (nigelmegitt) [inline spec/status.xml]: Minor editorial point - this reads strangely "An implementation report is available at implementation report documenting that..." Suggest changing to: 

```
<p>The <loc href="https://www.w3.org/wiki/TTML/TTML2ImplementationReport">implementation
report</loc> documents that, for each feature introduced in this
```

Comment (skynavga) [reply spec/status.xml]: ok; also, where can I find a link to use for AC feedback?

File: spec/status.xml [hunk 2]
```diff
@@ -19,72 +19,24 @@ href="mailto:public-tt@w3.org">public-tt@w3.org</loc> (<loc
...
-without having to publish a new Candidate Recommendation omitting the
-feature.</p>
-
-<p>A cumulative summary of all changes applied to this version since
-the current (TTML1, 2nd Edition) Recommendation was published is
-available at <loc href="ttml2-changes.html">Timed Text Markup Language
-2 (TTML2) Change Summary</loc>.  An abbreviated list of changes
-affecting language syntax can be found at
+with a subject line starting with <code>[ttml2]</code>.
+The deadline for AC Representative feedback is 01 November 2018.</p>
```
Comment (nigelmegitt) [inline spec/status.xml]: I think Pubrules wants an actual link to an actual feedback mechanism for the AC. I used https://github.com/w3c/ttml1/pull/367/files#diff-7a4f57e97ab443cec920609ffd9d4681R18 and it seems to satisfy it, though Link Checker complains it is behind an ACL.

**Comment by skynavga** [comment]
@nigelmegitt pubrules is happy now

File: spec/status.xml [hunk 3]
```diff
@@ -9,18 +9,19 @@ of this technical report can be found in the <loc
...
-be progressed to a W3C Recommendation. If you wish to make comments
-regarding this document, please file an <loc
-href="https://github.com/w3c/ttml2/issues">issue on GitHub</loc> or
-send email to <loc
-href="mailto:public-tt@w3.org">public-tt@w3.org</loc> (<loc
-href="mailto:public-tt-request@w3.org?subject=subscribe">subscribe</loc>,
-<loc
-href="http://lists.w3.org/Archives/Public/public-tt/">archives</loc>)
-with a subject line starting with <code>[ttml2]</code>.
-The deadline for AC Representative feedback is 01 November 2018.</p>
```
Comment (nigelmegitt) [inline spec/status.xml]: Pubrules good practice says to include the deadline for AC feedback as well as providing the link, I learned yesterday.

Comment (skynavga) [reply spec/status.xml]: It's already there. See the previous sentence.

**Comment by skynavga** [comment]
@nigelmegitt there are no unaddressed comments; please approve

File: spec/status.xml (hunk 3 above)
Comment (nigelmegitt) [reply spec/status.xml]: That looks like a deadline for general comments, not a deadline for AC review.

**Comment by skynavga** [comment]
@nigelmegitt it isn't the business of the SoTD to establish the AC review timeline; I followed the content found in a recently published PR [1] as a template; you are asking for more than is required or needed

[1] https://www.w3.org/TR/2018/PR-selectors-3-20180911/

**Comment by skynavga** [comment]
@nigelmegitt and it does so, and passes pubrules, so there is no need for a change

**Comment by nigelmegitt** [comment]
@skynavga it doesn't state the end of the AC review period.

**Comment by skynavga** [comment]
@nigelmegitt this is the kind of bothersome, unnecessary, time-consuming, pedantic, nit-picking that I have cautioned you about (repeatedly); if the pubrules checker is not complaining and the W3C is publishing PRs [1][2][3][4] that use this same language, then why are you complaining??? it is silly to say "submit comments by DATE" followed by "AC review deadline is DATE"; none of the PRs cited below use the phrase "AC review deadline" or similar;

if you insist on a change, then please make it directly to the PR branch and then approve for me to merge or go ahead and merge yourself

[1] https://www.w3.org/TR/2018/PR-selectors-3-20180911/
[2] https://www.w3.org/TR/2018/PR-css-fonts-3-20180814/
[3] https://www.w3.org/TR/2017/PR-page-visibility-2-20171017/
[4] https://www.w3.org/TR/2017/PR-requestidlecallback-20171010/

**Pull request merged**